tox.ini

```diff
@@ -18,11 +18,13 @@ extras = testing
 commands = pytest {posargs:--cov-config="{toxinidir}/tox.ini" --cov="{envsitepackagesdir}/tox" --timeout=180 tests}
 
 [testenv:docs]
-description = invoke sphinx-build to build the HTML docs, check that URIs are valid
+description = invoke sphinx-build to build the HTML docs and check that all links are valid
+whitelist_externals = sphinx-build
 basepython = python3.6
 extras = docs
-commands = sphinx-build -d "{toxworkdir}/docs_doctree" doc "{toxworkdir}/docs_out" --color -W -bhtml
-           sphinx-build -d "{toxworkdir}/docs_doctree" doc "{toxworkdir}/docs_out" --color -W -blinkcheck
+commands =
+    sphinx-build -d "{toxworkdir}/docs_doctree" doc "{toxworkdir}/docs_out" --color -W -bhtml
```

Comment:
this odd line break is some new style?

Reply:
Mor like an old styl of mine . If I have more than on entry in a list, I usually prefer to have them formatted like that.

Reply:
seems to me like adds no value, other than increasing the number of lines and changes in this file; and makes it inconsistent with other lines aligning content left to the =